types/granim/granim-tests.ts

```diff
@@ -0,0 +1,183 @@
+import { Granim } from 'granim';
```

Comment:
```suggestion
import Granim = require('granim');
```
This is for DT test only, users could use `esModuleInterop` to default to more common syntax